jekyll/_cci2/server/v4.1/air-gapped-installation/phase-5-configure-circleci-runner.adoc

```diff
@@ -0,0 +1,138 @@
+---
+version:
+- Server v4.x
+- Server Admin
+---
+= Phase 5 - Configuring CircleCI Runner
+:page-layout: classic-docs
+:page-liquid:
+:page-description: How to configure CircleCI Runner to run with CircleCI Server in an air-gapped environment.
+:icons: font
+:toc: macro
+:toc-title:
+
+[#install-circleci-cli]
+== 1. Install the latest CircleCI CLI release
+Retrieve and install the https://github.com/CircleCI-Public/circleci-cli/releases[latest CircleCI CLI release]. Determine the latest release, and then use the following command (substituting a different version to the one listed here if necessary):
+
+[source,shell]
+----
+wget https://github.com/CircleCI-Public/circleci-cli/releases/download/v0.1.22675/circleci-cli_0.1.22675_linux_amd64.tar.gz
+----
+
+[#copy-circleci-cli]
+== 2. Copy the CLI .tar.gz file
+Copy the CircleCI CLI `.tar.gz` file to the first machine designated to run the CircleCI runner in your air-gapped environment.
+
+[#extract-circleci-cli]
+== 3. Extract the CLI .tar.gz file
+On the air-gapped machine, extract the `.tar.gz` file:
+
+[source, bash]
+----
+tar -zxvf circleci-cli_0.1.22675_linux_amd64.tar.gz
+----
+
+[#cd-extracted-directory]
+== 4. Descend into the extracted directory
+Descend into the extracted directory.
+
+[source, bash]
+----
+cd circleci-cli_0.1.22675_linux_amd64
+----
+
+[#generate-user-token]
+== 5. Generate a personal API token
+In the app for your CircleCI server installation, generate a personal API token.
+
+To navigate to the page you need, use this URL format: `https://app.<airgapped-tld-of-your-circleci-server-installation>/settings/user/tokens`.
+
+[#export-user-token]
+== 6. Export the token
+On the air-gapped machine, export the user token, and the air-gapped accessible URL for your CircleCI server installation, as the following environment variables:
+
+NOTE: The `CIRCLECI_CLI_HOST` variable should *not* include the `app` subdomain of your installation that is present when visiting CircleCI server in a browser.
+
+NOTE: Adjust the protocol from https to http as necessary for your installation in the CIRCLECI_CLI_HOST environment variable.
+
+[source, bash]
+----
+export CIRCLECI_CLI_TOKEN=<generated-user-token>
+export CIRCLECI_CLI_HOST=https://<airgapped-tld-of-your-circleci-server-installation>
+----
+
+[#create-namespace]
+== 7. Create a runner namespace
+On the air-gapped machine, run the following command to create a namespace using your GitHub Enterprise username:
+
+[source,shell]
+----
+./circleci --skip-update-check namespace create <github-enterprise-username> github <github-enterprise-username>
+----
+
+[#create-resource-class]
+== 8. Create a resource class
+On the air-gapped machine, run the following command to create a resource class with your GitHub enterprise username:
+
+[source,shell]
+----
+./circleci --skip-update-check runner resource-class create <github-enterprise-username>/default "Default Runners" --generate-token
```

Comment:
@circlery in this command do they only need to sub their GHE username? Or is "Default Runners" something that will be customer-specific?